### PR TITLE
Removing old theme from App-kit components

### PR DESCRIPTION
**Pull request by chloe0592**
This PR is a cleanup after release of the new theme.

- Old illustrations were deleted and replaced with new ones.
- Setting the new theme `recolouring` as `default` one.
- Removed old theme overrides.
- Removed old theme code-blocks.

Note: **Percy fails** are expected for now. I will need to update UI-Kit dependencies after [this PR](https://github.com/commercetools/ui-kit/pull/2785) is merged and released in order for the new theme tokens to be implemented also in App-Kit.

### CODE REVIEW

**Review by CarlosCortizasCT — COMMENTED** [review]
Looks good to me but I hold my approval until `ui-kit` is updated.

Thanks Klaudija 🙇

**Review by ddouglasz — APPROVED** [review]
Looks good to me.

I believe the VRT will be taken care of then the uikit part of this task is merged. 
Thank you, Klaudija 🙌🏽

**Comment by chloe0592** [comment]
@CarlosCortizasCT, @ddouglasz could I, please, get one more look from you? 
I updated the UI-Kit dependencies and all looks good to me now.

**Comment by ddouglasz** [comment]
All good from my end 💯

File: packages/application-components/src/theming.ts [hunk 1]
```diff
@@ -8,33 +8,6 @@ import {
 // https://github.com/commercetools/merchant-center-application-kit/pull/3353
 export const themesOverrides = {
   default: {
```
Comment (CarlosCortizasCT) [inline packages/application-components/src/theming.ts:10]: After reviewing [the PR](https://github.com/commercetools/merchant-center-application-kit/pull/3414/files#diff-e98b3ff7bf77917ad89bcfae2e36182b3fcbcd90fbb148ae21fcfaa93b4ac30f) where we introduced the recolouring in `app-kit` and looking for usages of the remaining tokens here, I found there's a bunch of them that are use only in one place. For those ones, as we did in `ui-kit`, we should remove them and moved their values to the only place where they are needed.

Comment (chloe0592) [reply packages/application-components/src/theming.ts]: Thanks @CarlosCortizasCT for the input! I made some changes here - [3b8650a](https://github.com/commercetools/merchant-center-application-kit/pull/3497/commits/3b8650a3408847a15d3fa790816717fb35d147a0). Let me know if that's good now.

**Review by CarlosCortizasCT — APPROVED** [review]
Looks good to me know 💯 

Thanks a lot, Klaudija 🙇

**Pull request merged**